src/editor/map/map_context.cpp

```diff
@@ -155,71 +155,72 @@ map_context::map_context(const game_config_view& game_config, const std::string&
 	}
 
 	// 1.0 Pure map data
-	boost::regex rexpression_map_data(R"""(map_data\s*=\s*"(.+?)")""");
-	boost::smatch matched_map_data;
-
-	if(!boost::regex_search(
-		file_string, matched_map_data, rexpression_map_data, boost::regex_constants::match_not_dot_null)
-	) {
+	if(filename.rfind(".map")+4 == filename.size()) {
```

Comment:
While string::ends_with() won't be available until C++20, we already have filesystem::ends_with().